docs/guides/tutorials/getting_started/create_your_first_expectations.rst

```diff
@@ -3,44 +3,67 @@
 
 .. admonition:: Admonition from Mr. Dickens.
 
     "Take nothing on its looks; take everything on evidence. There's no better rule."
 
-The CLI will help you create your first Expectations. You can accept the defaults by typing [Enter] twice:
+The CLI will help you create your first Expectation Suite. :ref:`expectation_suites` are simply collections of Expectations. In order to create a new suite, we will use the ``scaffold`` command to create an Expectation Suite called ``taxi.demo`` with the help of a built-in profiler. Type the following into your terminal:
 
 .. code-block:: bash
 
-    Would you like to profile new Expectations for a single data asset within your new Datasource? [Y/n]: 
-    
-    Would you like to:
-        1. choose from a list of data assets in this datasource
-        2. enter the path of a data file
-    : 1
-    
-    Which data would you like to use?
-        1. npidata_pfile_20200511-20200517 (file)
-        Don't see the name of the data asset in the list above? Just type it
+    great_expectations suite scaffold taxi.demo
+
+You will see the following output:
+
+.. code-block:: bash
+
+    Heads up! This feature is Experimental. It may change. Please give us your feedback!
+
+    Which table would you like to use? (Choose one)
+        1. yellow_tripdata_sample_2019_01 (table)
+        2. yellow_tripdata_staging (table)
+        Do not see the table in the list above? Just type the SQL query
     : 1
-    
-    Name the new Expectation Suite [npidata_pfile_20200511-20200517.warning]: 
-    
-    Great Expectations will choose a couple of columns and generate expectations about them
-    to demonstrate some examples of assertions you can make about your data.
-    
-    Great Expectations will store these expectations in a new Expectation Suite 'npidata_pfile_20200511-20200517.warning' here:
-    
-      file:///home/ubuntu/example_project/great_expectations/expectations/npidata_pfile_20200511-20200517/warning.json
-    
-    Would you like to proceed? [Y/n]: 
-    
-    Generating example Expectation Suite...
-    
-    Done generating example Expectation Suite
+
+
+Which data should I choose when creating Expectations?
+====================
+
+In this example, we want to build an Expectation Suite based on what we know about our taxi data: each taxi ride has a ``passenger_count`` between 1 and 6. We then want to use that Expectation Suite to validate any future data that is loaded into the ``staging`` table. Hence, we choose the ``yellow_tripdata_sample_2019_01`` table when creating the new Expectation Suite, which we will then use to validate the ``yellow_tripdata_staging`` table **in a later step**. Makes sense, right?
```

Comment:
Again, I *love* the earlier introduction of batches. It feels *so* much more real.